HunterPie.UI/Architecture/Converters/WirebugStateToVisibilityConverter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using HunterPie.Integrations.Datasources.MonsterHunterRise.Entity.Enums;
+using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace HunterPie.UI.Architecture.Converters;
+
+public class WirebugStateToVisibilityConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        var wirebugState = (WirebugState)value;
+        return wirebugState == WirebugState.Blocked
+            ? Visibility.Visible
+            : Visibility.Collapsed;
```

Comment:
Not really a required change, but usually I'd code it like this:

```suggestion
        if (value is not WirebugState state)
            throw new ArgumentException($"Argument must be of type {nameof(WirebugState)} but got {value?.GetType()}");
            
        return state == WirebugState.Blocked
            ? Visibility.Visible
            : Visibility.Collapsed;
```

I'd personally only hard cast things if I know for sure the object is going to be that type, since converters can be used anywhere you don't really know if the type is going to always be `WirebugState`. Converters are also called in design time, so it's nice to know what exactly happened during design time.